src/express-tooling.ts

```diff
@@ -105,27 +114,39 @@ function handleOAuthRequestMiddleware(options: MiddlewareOptions): RequestHandle
     const authHeader = getHeaderValue(req, AUTHORIZATION_HEADER_FIELD_NAME);
 
     if (!authHeader) {
+      if (logger) {
+        logger.error('No authorization field in header');
+      }
       rejectRequest(res);
       return;
     }
 
     const accessToken = extractAccessToken(authHeader);
     if (!accessToken) {
+      if (logger) {
+        logger.error('No access_token field in header');
```

Comment:
I rather see here a warn or info.
Furthermore the error message should something like `'access_token is empty'` I guess